### PR TITLE
Add `AcceptableMediaTypes` to `organizationalBrandingProperties` media stream properties

### PR DESCRIPTION
**Pull request by irvinesunday**
Fixes https://github.com/microsoftgraph/msgraph-metadata/issues/466

### CODE REVIEW

File: transforms/csdl/preprocess_csdl.xsl [hunk 1]
```diff
@@ -171,6 +171,30 @@
...
+                         edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='organizationalBrandingProperties']/edm:Property[@Name='headerLogo']|
+                         edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='organizationalBrandingProperties']/edm:Property[@Name='squareLogo']|
+                         edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='organizationalBrandingProperties']/edm:Property[@Name='squareLogoDark']">
+        <xsl:copy>
+            <xsl:copy-of select="@* | node()" />
+            <Annotation Term="Org.OData.Core.V1.AcceptableMediaTypes">
+                <Collection>
+                    <String>image/bmp</String>
+                    <String>image/jpg</String>
+	                <String>image/jpeg</String>
```
Comment (calebkiage) [inline transforms/csdl/preprocess_csdl.xsl:188]:
```suggestion
	            <String>image/jpeg</String>
```

File: transforms/csdl/preprocess_csdl.xsl [hunk 2]
```diff
@@ -171,6 +171,30 @@
...
+                    <String>image/gif</String>
+	                <String>image/vnd.microsoft.icon</String>
```
Comment (calebkiage) [inline transforms/csdl/preprocess_csdl.xsl:190]:
```suggestion
	            <String>image/vnd.microsoft.icon</String>
```

**Review by andrueastman — APPROVED** [review]

File: transforms/csdl/preprocess_csdl.xsl (hunk 1 above)
Comment (irvinesunday) [reply transforms/csdl/preprocess_csdl.xsl:188]: I promise they are aligned 😄, view the PR on the github.dev web-based editor (press the . key)

File: transforms/csdl/preprocess_csdl.xsl (hunk 2 above)
Comment (irvinesunday) [reply transforms/csdl/preprocess_csdl.xsl:190]: This one is aligned too.

**Review by peombwa — APPROVED** [review]

**Pull request merged**